cluster_tools/cluster_tools/schedulers/slurm.py

```diff
@@ -111,7 +151,7 @@ def get_max_array_size():
             "scontrol show config | sed -n '/^MaxArraySize/s/.*= *//p'"
         )
         if exit_code == 0:
-            max_array_size = int(stdout.decode("utf8"))
```

Comment:
this wasn't necessary?

Reply:
I refactored the `call` function above to do this instead, since it was decoded for all usages.